.github/workflows/spec.yml

```diff
@@ -120,6 +120,10 @@ jobs:
           buildevents cmd $TRACE_ID $STEP_ID 'bundle env' -- bundle env
           echo ::endgroup::
 
+      - name: Create task helper symlink
+        run: |
+          ln -s "${PWD}/spec/fixtures/modules/ruby_task_helper" ..
+
```

Comment:
FYI, I created https://github.com/puppetlabs/pdk-templates/pull/461 to handle this in the future without regression.

Reply:
Merged. I updated `.sync.yml` to enable the inclusion of this code.